### PR TITLE
[16.0][FIX] rma: _get_sequence_values signature

### PR DESCRIPTION
**Pull request by chienandalu**
Respect the orignal method signature

https://github.com/odoo/odoo/blame/6bb68d9f3263c2154eb87ec22559d142e880c084/addons/stock/models/stock_warehouse.py#L1017

please review  @pedrobaeza @victoralmau 

cc @Tecnativa

### CODE REVIEW

**Review by victoralmau — APPROVED** [review]

File: rma/models/stock_warehouse.py [hunk 1]
```diff
@@ -56,8 +56,8 @@ def _get_rma_location_values(self):
             "location_id": self.env.ref("rma.stock_location_rma").id,
         }
 
-    def _get_sequence_values(self):
-        values = super()._get_sequence_values()
+    def _get_sequence_values(self, name=False, code=False):
+        values = super()._get_sequence_values(name, code)
```
Comment (pedrobaeza) [inline rma/models/stock_warehouse.py]: Always pass the keyword arguments as keyword:

```suggestion
        values = super()._get_sequence_values(name=name, code=code)
```

**Comment by pedrobaeza** [comment]
I have noted that uninstalling the module, the sequences remain. Is there any cleanup code for that in standard?

**Comment by chienandalu** [comment]
> I have noted that uninstalling the module, the sequences remain. Is there any cleanup code for that in standard?

Not that I'm aware of. I guess you don't even need to uninstall, but just remove a warehouse and that sequences dirt will stay there forever

**Comment by pedrobaeza** [comment]
hahaha: https://github.com/odoo/odoo/pull/134032

**Comment by chienandalu** [comment]
Oh :)

**Pull request merged**